sdk/appconfiguration/azure-appconfiguration-provider/README.md

```diff
@@ -110,6 +110,37 @@ Once the provider is refreshed, the configurations can be accessed as normal. An
 
 For additional info check out [Dynamic Refresh](https://learn.microsoft.com/azure/azure-app-configuration/enable-dynamic-configuration-python) on MS Learn.
 
+## Selecting Feature Flags
+
+You can enable loading of feature flags by setting `feature_flags_enabled` to `True`. By default all feature flags with no label are loaded.
+
+```python
+config = load(
+    connection_string=connection_string,
+    feature_flag_enabled=True,
+)
+```
+
+If you want to load feature flags with a specific label you can use `feature_flag_selectors` and provide a list of `SettingSelector`s to filter the feature flags.
+
+```python
+config = load(
+    connection_string=connection_string,
+    feature_flag_enabled=True,
+    feature_flag_selectors=[SettingSelector(key_filter="*", label_filter="dev")]
+)
+```
+
+Feature flags can be refreshed in a similar way to configurations. By setting `feature_flag_refresh_enabled` to `True` and calling `refresh` on the provider.
+
+```python
+config = load(
+    connection_string=connection_string,
+    feature_flag_enabled=True,
+    feature_flag_refresh_enabled=True,
+)
+```
+
```

Comment:
Can you add the code to trigger refresh of feature flags?

Reply:
It is there, but in the other section. This one is now removed.